sdcm/nemesis.py

```diff
@@ -4241,20 +4241,20 @@ def disrupt_add_remove_mv(self):
                     raise UnsupportedNemesis(
                         'A supported column for creating MV is not found. nemesis can\'t run')
                 column = f'"{column}"'
-                self.log.info("Stopping Scylla on node %s", self.target_node.name)
-                self.target_node.stop_scylla()
-                InfoEvent(message=f'Create a materialized-view for table {ks_name}.{base_table_name}').publish()
-                try:
-                    with EventsFilter(event_class=DatabaseLogEvent,
-                                      regex='.*view - Error applying view update.*',
-                                      extra_time_to_expiration=30):
+                with EventsFilter(event_class=DatabaseLogEvent,
+                                  regex='.*Error applying view update.*',
+                                  extra_time_to_expiration=180):
+                    self.log.info("Stopping Scylla on node %s", self.target_node.name)
+                    self.target_node.stop_scylla()
```

Comment:
@yarongilor if we are getting errors cause of stop in this nemesis, it mean we have other MVs (or left overs from this nemesis, or from the main load), and this failure messages can happen anytime scylla is stopped.

so this change doesn't really avoid those from happening across the board.

Reply:
@fruch , i don't think it's the case here, because we don't see such errors across the board.
It looks like a matter of a race condition of SCT filter and DB logs.
So if it sounds better, we can simply sleep a minute after applying the filter, before creating the MV.

Reply:
if those exceptions are not related to the creation of view, why do you need the stop inside this context ? 
and how adding a minute sleep would change anything if the failures are during the stop ?

I don't see any evidence here for any "race" condition with the event system...